custom_components/zonneplan_one/const.py

```diff
@@ -65,7 +65,7 @@ class ZonneplanSensorEntityDescription(SensorEntityDescription):
             name="Zonneplan current tariff group",
         ),
         "current_tariff": ZonneplanSensorEntityDescription(
-            key="summary_data.price_per_hour.0.price",
+            key="summary_data.price_per_hour.24.price",
```

Comment:
Are these values always available under these keys?

Reply:
Yes. I just checked, the fix also works for the morning (before 1PM). Have a look at the current price data attached below... I would like to have it a bit more robust though instead of relying on list indices ;). That would also benefit #22 which I also would like a lot. I hope I can look into that somewhere in the next weeks (unless you are planning so yourself ;)).

[Zonneplan hourly rates (before 1PM).txt](https://github.com/fsaris/home-assistant-zonneplan-one/files/10039507/Zonneplan.hourly.rates.before.1PM.txt)

Reply:
I've tested the changes. The forecast is fixed however I'm getting a "Unknown" for the current tariff when changing to 24.

Reply:
I tested the changes as well, for me both the forcast and the current tariff is fixed.

Reply:
Current seems to be auto corrected after an hour. So all good!

Reply:
> I've tested the changes. The forecast is fixed however I'm getting a "Unknown" for the current tariff when changing to 24.

That would be strange imho, given the fact that indices 25-32 do work :-). I would rather be scared by the highest index than the lower...